ushahidi/platform-client
# Update dependencies

This pull request makes the following changes:
- Updates minor versions of libraries
- Updates to latest versionmini-css-extract-plugin
- Update to a new version of chart.js + its adapter 
- Updates lockfiles
- Add node-version for the client

Testing checklist:
- Run npm run install:all
- [ ] It should install without problem
- Run npm run serve
- [ ] It should serve the app on localhost:3000


Fixes ushahidi/platform# .

Ping @ushahidi/platform
